src/command.js

```diff
@@ -11,6 +11,12 @@ import pkg from '../package.json';
 
 // used to deserialize regular expression strings
 const RE_REGEXP = /^\/(.+)\/(\w+)?$/;
+const WAIT_FOR_TIMEOUT = 5000;
+
+// remove stack traces from any eval error
+function normalizeEvalError(error) {
+  return error.replace(/^Error:\s(.*?)\n\s{4}at\s.*$/s, '$1');
```

Comment:
In case a real error is thrown, we should add a string check like the other catch has for this:

```suggestion
  if (typeof error !== 'string') return error;
  return new Error(error.replace(/^Error:\s(.*?)\n\s{4}at\s.*$/s, '$1'));
```